cmd/playlist.go

```diff
@@ -202,6 +213,44 @@ func addto(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func listPlaylists(cmd *cobra.Command, args []string) error {
+	// current user
+	user, err := client.CurrentUser()
+	if err != nil {
+		return err
+	}
+	fmt.Println("User: ", user.DisplayName)
```

Comment:
```suggestion
	fmt.Printf("User: %s\n", user.DisplayName)
```

Reply:
I don't have a strong feeling one way or the other. I was simply duplicating the existing style. It should probably be consistent with the other methods, whichever way you want to do it.